### PR TITLE
AT 01.06 | Freestyle project > Rename project > add 4 tests

### PR DESCRIPTION
**Pull request by AnaZaleski**
https://trello.com/c/UoOY3bAG
https://trello.com/c/Z8WNVCUG
https://trello.com/c/NbiIkOu1
https://trello.com/c/6K463lOW

### CODE REVIEW

**Comment by AnaZaleski** [comment]
[AT 01.06 | Freestyle project > Rename project > Verify new name is present on Dashboard Page after Rename from Dashboard List](https://trello.com/c/6K463lOW/1221-at-0106-freestyle-project-rename-project-verify-new-name-is-present-on-dashboard-page-after-rename-from-dashboard-list)

**Comment by AnaZaleski** [comment]
[AT 01.06 | Freestyle project > Rename project > Verify rename project from the Dashboard List](https://trello.com/c/UoOY3bAG/1219-at-0106-freestyle-project-rename-project-verify-rename-project-from-the-dashboard-list)

**Comment by AnaZaleski** [comment]
[AT 01.06 | Freestyle project > Rename project > Verify new name is present on Dashboard Page after Rename from Project page](https://trello.com/c/NbiIkOu1/1220-at-0106-freestyle-project-rename-project-verify-new-name-is-present-on-dashboard-page-after-rename-from-project-page)

**Comment by AnaZaleski** [comment]
[AT 01.06 | Freestyle project > Rename project > Verify rename project from the Project page](https://trello.com/c/Z8WNVCUG/1207-at-0106-freestyle-project-rename-project-verify-rename-project-from-the-project-page)

**Review by GelikaUS — APPROVED** [review]

**Review by VictoriaLema — APPROVED** [review]

File: src/test/java/school/redrover/FreestyleProject8Test.java [hunk 1]
```diff
@@ -100,4 +102,67 @@ public void testEditDescription () {
...
+
+        WebElement renameProjectButton = getDriver()
+                .findElement(By.xpath("//*[@href = '/job/NewProject/confirm-rename']"));
+        renameProjectButton.click();
+
+        WebElement newNameInputField = getDriver().findElement(By.xpath("//*[@name = 'newName']"));
+        newNameInputField.clear();
+        newNameInputField.sendKeys("OldProject");
+
+        WebElement renameButton = getDriver().findElement(By.xpath("//*[@name = 'Submit']"));
```
Comment (VictoriaLema) [inline src/test/java/school/redrover/FreestyleProject8Test.java:118]: тут лучше бы подождать с getWait

**Pull request merged**